Reyes_PaulAngelo/works/Countries.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Countries of the World</title>
+    <link rel="stylesheet" href="./css_country.css" />
+  </head>
+  <body>
+    <div class="container">
+      <h1>Countries of the World</h1>
+      <input
+        type="text"
+        id="countryInput"
+        placeholder="Enter country name..."
+      />
+      <button onclick="searchCountry()">Search</button>
+      <div class="country-details" id="countryDetails"></div>
+      <div class="same-region-countries" id="sameRegionCountries"></div>
```

Comment:
IDs should be written in snake case. All lower case separated by underscore.

Reference: https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_5